app/main/helpers/diff_tools.py

```diff
@@ -1,6 +1,6 @@
 import difflib
 try:
-    from itertools import izip_longest
+    from itertools import zip_longest
 except ImportError:
     from itertools import zip_longest as izip_longest
```

Comment:
```suggestion
```
Should we remove the `ImportError`? or at least change `izip_longest`?

Reply:
We already use zip_longest in line 102 so it should declare from itertools import zip_longest. No where in the file uses izip_longest